server/proxy/pf_config.c

```diff
@@ -30,195 +30,143 @@
 
 #define TAG PROXY_TAG("config")
 
-#define CHANNELS_SEPERATOR ","
+static inline BOOL pf_config_get_bool(wIniFile* ini, const char* section, const char* key)
+{
+	return IniFile_GetKeyValueInt(ini, section, key);
+}
 
-wArrayList* parse_string_array_from_str(const char* str)
+static BOOL pf_config_get_uint16(wIniFile* ini, const char* section, const char* key, UINT16* port)
```

Comment:
port -> value

Reply:
just a second after you comment, renamed to result